snippets/python/get_nb_images.py

```diff
@@ -7,20 +7,22 @@
 nbmaster = ""
```

Comment:
Naming of this file and get_nb_catalog_images.pl should be consistent.

Reply:
done